### PR TITLE
Embed api-reference properly on the docs website. 

### PR DESCRIPTION
**Pull request by themr0c**
* Embed API-reference properly on the docs website.
* Add devfile and tools for more user-friendly build.

Screenshot from 2020-09-18 13-32-18

### CODE REVIEW

File: antora-playbook.yml [hunk 1]
```diff
@@ -10,8 +10,4 @@ ui:
   bundle:
     url: https://gitlab.com/antora/antora-ui-default/-/jobs/artifacts/master/raw/build/ui-bundle.zip?job=bundle-stable
     snapshot: true
-  supplemental_files:
-  - path: ui.yml
-    contents: |
-      static_files: [ .nojekyll ]
-  - path: .nojekyll
+  supplemental_files: ./supplemental-ui
```
Comment (sleshchenko) [inline antora-playbook.yml:13]:
```suggestion
  supplemental_files: ./supplemental-ui

```

File: devfile.yaml [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+    type: dockerimage
+    image: 'docker.io/mrksu/newdoc'
+    memoryLimit: 512M
+    mountSources: true
+    command:
+      - tail
+    args:
+      - '-f'
+      - /dev/null
+  - id: testthedocs/vale/latest
```
Comment (sleshchenko) [inline devfile.yaml:50]: I would vote for placing plugins above dockerimages since they usually take less space

File: devfile.yaml [hunk 3]
```diff
@@ -0,0 +1,69 @@
+# chectl devfile:generate --name=che-docs --git-repo=https://github.com/eclipse/che-docs --dockerimage=docker.io/antora/antora --command=./run_antora.sh
```
Comment (sleshchenko) [inline devfile.yaml:1]: Isn't it enough to have devfile.yaml? Since you also adds empty devfile.yml

**Review by sleshchenko — APPROVED** [review]

File: devfile.yaml (hunk 3 above)
Comment (themr0c) [reply devfile.yaml:1]: empty devfile.yml was a mistake ...

File: devfile.yaml (hunk 2 above)
Comment (themr0c) [reply devfile.yaml:50]: ok

**Comment by themr0c** [comment]
Screenshot from 2020-09-18 15-20-41

**Comment by themr0c** [comment]
NB: I don't have the powers to merge this.

**Review by l0rd — APPROVED** [review]

**Pull request merged**